pkgs/development/python-modules/gtimelog/default.nix

```diff
@@ -42,18 +78,16 @@ buildPythonPackage rec {
     longDescription = ''
       GTimeLog is a small time tracking application for GNOME.
       It's main goal is to be as unintrusive as possible.
-
       To run gtimelog successfully on a system that does not have full GNOME 3
       installed, the following NixOS options should be set:
       - programs.dconf.enable = true;
       - services.gnome.gnome-keyring.enable = true;
-
       In addition, the following packages should be added to the environment:
       - gnome.adwaita-icon-theme
       - gnome.dconf
```

Comment:
`gnome.dconf` has long been renamed to `dconf`. But `programs.dconf.enable = true;` is now sufficient.